index.js

```diff
@@ -83,7 +135,21 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  let matchingId = {};
+  if (movies.length === 0) {
+    return null;
+  } 
+  for (let i = 0; i < movies.length; i++){
+    if (movies[i].imdbID === id){
+      matchingId = movies[i];
+    }
+  }
+  if (Object.keys(matchingId).length === 0){
+    return null;
+  }
+  return matchingId;
```

Comment:
Nice `Object.keys` call!  However why not just return `matchingId` on line 145?   Else return null.